### PR TITLE
e/950 - Footer set to absolute bottom on small pages

### PR DESCRIPTION
**Pull request by gingerbenw**
test height of page and set accordingly, slightly restructured footer column sizing

### CODE REVIEW

**Comment by gingerbenw** [comment]
Solved #950

**Comment by gingerbenw** [comment]
Hey Shane, I've got the code working AOK for this, but if the homepage takes to long to render, the footer ends up in the wrong place. Current workaround is to set a timeout, unless you had a better alternative? document ready does not seem to solve the issue.

Please see last few lines of assets/js/fixFooter.js

File: _pages/faq.md [hunk 1]
```diff
@@ -26,18 +26,19 @@ Yes. We have deliberately developed an open source solution to reporting UK SDGs
...
+	})
+	$('.roleHeader').click(function () {
+		$(this).nextUntil('h2').stop(true, true).toggle()
+			$(window).trigger('resize')
+	}).nextUntil('h2').hide();
+	$('.roleHeader').keypress(function (e) {
+		if (e.which == 13 || e.which == 32) { //Enter or space key pressed
+			$(this).trigger('click');
+		}
+	})
```
Comment (shaneporter) [inline _pages/faq.md]: Should this file be included in this PR?

**Comment by gingerbenw** [comment]
No longer required

**Pull request closed without merging**